.github/workflows/reusable_build_image.yaml

```diff
@@ -7,6 +7,9 @@ on:
       sdp-version:
         required: true
         type: string
+      image-namespace:
+        type: string
+        default: "sdp"
```

Comment:
I do wonder if the default should be `stackable`, and then only those promoted into the platform can use `sdp`?.

We can consider that change later though, as it means changing all of the other workflows.